src/extension.ts

```diff
@@ -107,13 +94,13 @@ class ProbeRSDebugAdapterServerDescriptorFactory implements vscode.DebugAdapterD
 				open: () => {
 					let windowIsOpen = true;
 					session.customRequest("rttWindowOpened", { channelNumber, windowIsOpen }).then((response) => {
-						logToConsole("DEBUG: probe-rs: RTT Window opened, and ready to receive RTT data on channel" + JSON.stringify(channelNumber, null, 2));
+						logToConsole("probe-rs-debug: RTT Window opened, and ready to receive RTT data on channel" + JSON.stringify(channelNumber, null, 2));
```

Comment:
This was not introduced here, but I would prefer if templates/format literals would be used instead of string concatenation :) Alos, the prefix here could be made the same constant as above to be consistent across the codebase :)

Reply:
Done :)